src/server/services/wms/qgswmsgetprint.cpp

```diff
@@ -39,6 +39,10 @@ namespace QgsWms
     {
       contentType = "image/png";
     }
+    else if ( format == QgsWmsParameters::JPG )
+    {
+      contentType = "image/jpeg";
```

Comment:
If `contentType` is a `QString` you should use `QStringLiteral("image/jpeg");` also for the others.

Reply:
Changes done